package.json

```diff
@@ -7,17 +7,20 @@
     "build": "build-figma-plugin --typecheck --minify",
     "watch": "build-figma-plugin --typecheck --watch"
   },
-  "author": "Vilhelm Sjölander",
+  "author": {
+    "name": "Green Design System",
+    "email": "greendesignsystem@seb.se"
+  },
   "license": "MIT",
   "devDependencies": {
-    "@create-figma-plugin/build": "^2.6.1",
-    "@create-figma-plugin/tsconfig": "^2.6.1",
-    "@figma/plugin-typings": "^1.76.0",
+    "@create-figma-plugin/build": "^3.1.0",
```

Comment:
I would suggest pinning your dependencies (and setting up Dependabot to manage updates). This gives you more control over dependencies' and subdependencies' versions.

Reply:
I will add dependabot, not sure why it hasn't been added.

Regarding pinning: don't you miss out on secority and feature updates that aren't breaking?